### DISCUSSION TITLE
exploring-dotnet-6-part-4-building-a-middleware-pipeline-with-webapplication/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#15] [Announcements]
# exploring-dotnet-6-part-4-building-a-middleware-pipeline-with-webapplication/

In this post I look at some of the code behind WebApplication, and focus on how the middleware and endpoints are configured

https://andrewlock.net/exploring-dotnet-6-part-4-building-a-middleware-pipeline-with-webapplication/

**Comment by DavidLazarescu** [comment]
Thanks for your articles, they are very helpful.

[a comment by an automated account is not shown]

**Comment by eagrzn** [comment]
It's so sad that you can't get rid of `DeveloperExceptionPageMiddleware`!
